aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -46,26 +56,29 @@ Resources:
           files:
             '/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json':
               content: !Sub |
-                {
+                { 
+                  "agent": {
+                    "debug": true
```

Comment:
I don't think we should turn on debug = true for all users of this template.

Reply:
resolved